ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -208,7 +248,11 @@ ssh ${SSH_ARGS} root@${bastion} "
      ansible-playbook -i ansible/inventory/$LAB_CLOUD.local ansible/hv-setup.yml -v | tee /tmp/ansible-hv-setup-$(date +%s)
```

Comment:
We also need to run `hv-setup.yml` and `hv-vm-create.yml` to create the VMs

Reply:
oh, I missed it. Thanks @mcornea